webpack.config.prod.js

```diff
@@ -6,14 +6,24 @@ module.exports = merge(commonConfig, {
   devtool: "source-map",
   mode: "production",
   optimization: {
+    splitChunks: {
+      cacheGroups: {
+        default: false, // disable default cache group
+        vendors: {
+          test: /[\\/]node_modules[\\/]/,
+          chunks: "all",
+          enforce: true, // always create a chunk
+        },
+      },
+    },
     minimizer: [
       new UglifyJsPlugin({
         uglifyOptions: {
-          mangle: false
+          mangle: false,
```

Comment:
I see that you disabled mangling  "IE 11 support. (#1678)". Could we add a comment about this in the code ?

Reply:
Mangling was already in the previous configuration, the refactor dit not change that I believe. 
You cannot have simple mangling because, currently,  we rely on components names for rails send which component should be initialized (you can add this as a comment on that line...). We could eventually try adding "name" exceptions to mangle, but might not be straightforward (i wonder if there is a way to make component names an exception).

Here I was asking about enable mangling for the third party code only, but not sure how easy that is either.

Reply:
Ah because of `react_component`? Good explanation.